database/migrations/2023_01_30_173822_add_scopes_to_existing_oauth_access_tokens.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use App\Providers\AuthServiceProvider;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void {
+        Schema::table('oauth_access_tokens', function(Blueprint $table) {
+            DB::table('oauth_access_tokens')->update(['scopes' => '["' . implode('","', array_keys(AuthServiceProvider::$scopes)) . '"]']);
```

Comment:
I would not give permission to access everything by default. Would rather exclude the `extra` scopes? For future keys, the user must and can then decide for himself.